packages/push/examples/smallest/main.rs

```diff
@@ -111,8 +109,6 @@ fn main() -> Result<()> {
         .into_collection_generator(population_size)
         .sample(&mut rng);
 
-    ensure!(population.is_empty().not());
```

Comment:
Why was this removed?

Reply:
I'm happy to put it back if you think it's useful. I think it was a leftover from early on when I was writing the first `main` examples, and then I just wanted to make sure that things were working. It seemed like a "leftover" to me, so I removed it.

Reply:
I think this was probably a useful check, at least it seemed that way to me - I would just leave it in.